Externals/wxWidgets3/CMakeLists.txt

```diff
@@ -895,4 +895,10 @@ else()
         message(FATAL_ERROR "wxWidgets in Externals is not compatible with your platform")
 endif()
 
+if (MSVC)
+	add_definitions(/W0)
```

Comment:
If you don't use what I suggested, please at least use `add_compile_options()` or even `target_compile_options(lib PRIVATE ...)`